src/components/TreeSelect/Dropdown.tsx

```diff
@@ -0,0 +1,90 @@
+import { useState } from "react";
+import styled from "@emotion/styled";
+import Icon from "@leafygreen-ui/icon";
+import { uiColors } from "@leafygreen-ui/palette";
+
+const { gray } = uiColors;
+
+interface RenderProps {
+  isVisible: boolean;
+  setOptionsLabel?: React.Dispatch<React.SetStateAction<string>>;
+}
+
+interface Props {
+  inputLabel: string;
+  "data-cy": string;
+  width?: string;
+  render: ({
+    getDropdownProps,
+  }: {
+    getDropdownProps: () => RenderProps;
+  }) => JSX.Element;
+}
+
+// including a TreeDataEntry with value = "all"
+// will serve as the 'All' button
+export const Dropdown: React.FC<Props> = ({
+  inputLabel, // label for the select
+  "data-cy": dataCy, // for testing only
+  width,
+  render,
```

Comment:
can we add a jest test for Dropdown that verifies it will render the output of the render function when it's provided